### ISSUE TITLE
Bug: Flags don't show up in language selector in Chromium

### ISSUE THREAD

**Issue by nzakas** [eslint/new.eslint.org#183]
### URL(s)

https://es.eslint.org
https://es.eslint.org/languages

### What did you do?

Scroll all the way to the bottom to the language picker.

### What did you expect to happen?

To see a flag next to "Spanish (ES)" as in Firefox:
![firefox-language-picker](https://user-images.githubusercontent.com/38546/160222623-3ebf2e66-4ec2-4d22-a382-320f7375421b.png)

### What actually happened?

I just saw "ES" instead of the flag. From Chrome:
![chrome-language-picker](https://user-images.githubusercontent.com/38546/160222635-2214ce51-7ff2-4440-b438-307791d4cedf.png)

### Participation

- [X] I am willing to submit a pull request for this issue.

### Additional comments

This appears on all language sites, not just Spanish.

**Comment by nzakas** [comment]
@SaraSoueidan do you have any idea what might be happening here?

**Comment by SaraSoueidan** [comment]
No, that’s odd. I just checked on my macOS and in my Windows VM and it seems that this is happening only on Chrome on Windows. I'm not sure why. I've never come across this before. 🤔

**Comment by SaraSoueidan** [comment]
Apparently it's a known Windows 10 issue: https://answers.microsoft.com/en-us/windows/forum/all/flag-emoji/85b163bc-786a-4918-9042-763ccf4b6c05

Do you get the same problem in Windows 11? I can't test on Win 11 because I don't have its VM yet.

**Comment by nzakas** [comment]
I haven’t upgraded to Windows 11, so I’ll not sure. It seems like there’s not a good solution here. Maybe worth filing a bug on Chrome.

**Comment by SaraSoueidan** [comment]
I think others have already done that. It seems to be a Windows issue more so than a Chrome issue.

**Comment by nzakas** [comment]
It works in Firefox on Windows, so it must be solvable.
